docs/src/components/code-block/actions.tsx

```diff
@@ -8,6 +8,37 @@ import pkg from '../../../package.json'
 import usePrevious from '../../hooks/use-previous'
 import styles from './styles.module.css'
 
+const CODESANDBOX_CSS_FILE = `
```

Comment:
i could make this cleaner....maybe use file-loader or something....but this was easy...